### DISCUSSION TITLE
ΓΕ2 - ΜΕΡΟΣ Β

### DISCUSSION THREAD

**Discussion by Katethoma** [sdy60/syllabus#43] [Q&A] [answered]
Καλησπέρα. Ήθελα να ρωτήσω τι ακριβώς ζητάτε στο ΜΕΡΟΣ Β, Ερώτημα 1 ?

**Comment by epidrome** [comment, marked as answer]
Το ζητούμενο είναι ένας συνδυασμός των Β1 και Δ από την ΓΕ1. δλδ. εντοπισμός προδιαγραφών και κριτηρίων ποιότητας της εφαρμογής στην επιστημονική βιβλιογραφία και όχι σε εμπορικά προϊόντα
